shub/config.py

```diff
@@ -33,6 +34,25 @@ def __init__(self):
         self.stacks = {}
         self.requirements_file = None
 
+    def _check_endpoints(self):
+        """Check the endpoints. Send warnings if necessary."""
+        for endpoint, url in self.endpoints.items():
+            parsed = urlparse.urlparse(url)
+            if parsed.netloc == 'staging.scrapinghub.com':
+                click.echo(
+                    'WARNING: Endpoint "%s" is still using %s. '
+                    'Please change the netloc to app.scrapinghub.com' % (
+                        endpoint, parsed.netloc,
+                    ),
```

Comment:
Could you add an explanation that `staging.scrapinghub.com` was dropped/deprecated/no longer supported to the message?